www/testlog.php

```diff
@@ -52,7 +55,7 @@
         'local' => isset($_REQUEST['local']) && $_REQUEST['local'],
         'body_class' => 'history',
         'page_title' => 'WebPageTest - Test History',
-
+        'filter' => $filterstr
```

Comment:
I know you have the regex above, but I'd just use `htmlspecialchars()` https://www.php.net/htmlspecialchars

Reply:
hey jeff since its being rendered on a blade template it should be going through this function already, should we add anyway?

![image](https://github.com/WPO-Foundation/webpagetest/assets/4400047/5ababf65-7bf2-4991-b639-7a15872a6134)

https://laravel.com/docs/10.x/blade

Reply:
oh that's right! nice catch. spaced the whole "this is on blade" thing.